specifyweb/frontend/js_src/lib/components/SkeletonLoaders/Form.tsx

```diff
@@ -4,7 +4,7 @@ import { Skeleton } from './Skeleton';
 
 export function FormSkeleton() {
   return (
-    <Skeleton.Root className="flex-col">
+    <Skeleton.Root className="flex-col flex-wrap overflow-auto">
```

Comment:
does `flex-wrap` make any difference here?
I though it doesn't change anything in most cases unless you have `flex-row`

Reply:
No you're right. Doesn't seem like it